core/primitives/src/views.rs

```diff
@@ -255,6 +290,9 @@ pub enum QueryRequest {
         #[serde(rename = "args_base64", with = "base64_format")]
         args: FunctionArgs,
     },
+    ViewCode {
+        account_id: AccountId,
+    },
```

Comment:
I would move it to after the `ViewAccount` as it semantically belongs there in my opinion.